chipsec/hal/acpi_tables.py

```diff
@@ -1928,14 +1934,22 @@ def parseMAP(self, tableLen, table_content):
 '''.format(tableLen, tableLen, nfitDeviceHandle, nvdimmPhysID, nvdimmRegionID, spaRangeStructureIndex, nvdimmControlRegionSz, nvdimmRegionSz, regionOffset, nvdimmPhysicalAddressRegionBase, interleaveStructIndex, interleaveWays, nvdimmStateFlags, reserve)
 
     def parseSPA(self, tableLen, table_content):
-        volitileMemGUID = [int('0x7305944f', 16), int('0xfdda', 16), int('0x44e3', 16), int('0xb1', 16), int('0x6c', 16), int('0x3f', 16), int('0x22', 16), int('0xd2', 16), int('0x52', 16), int('0xe5', 16), int('0xd0', 16)]
-        byteAddrPMGUID = [int('0x66f0d379', 16), int('0xb4f3', 16), int('0x4074', 16), int('0xac', 16), int('0x43', 16), int('0x0d', 16), int('0x33', 16), int('0x18', 16), int('0xb7', 16), int('0x8c', 16), int('0xdb', 16)]
-        nvdimmControlRegionGUID = [int('0x92f701f6', 16), int('0x13b4', 16), int('0x405d', 16), int('0x91', 16), int('0x0b', 16), int('0x29', 16), int('0x93', 16), int('0x67', 16), int('0xe8', 16), int('0x23', 16), int('0x4c', 16)]
-        nvdimmBlckDataWindowRegionGUID = [int('0x91af0530', 16), int('0x5d86', 16), int('0x470e', 16), int('0xa6', 16), int('0xb0', 16), int('0x0a', 16), int('0x2d', 16), int('0xb9', 16), int('0x40', 16), int('0x82', 16), int('0x49', 16)]
-        ramDiskVirtualDiskVolGUID = [int('0x77ab535a', 16), int('0x45fc', 16), int('0x624b', 16), int('0x55', 16), int('0x60', 16), int('0xf7', 16), int('0xb2', 16), int('0x81', 16), int('0xd1', 16), int('0xf9', 16), int('0x6e', 16)]
-        ramDiskVirtualCDVolGUID = [int('0x3d5abd30', 16), int('0x4175', 16), int('0x87ce', 16), int('0x6d', 16), int('0x64', 16), int('0xd2', 16), int('0xad', 16), int('0xe5', 16), int('0x23', 16), int('0xc4', 16), int('0xbb', 16)]
-        ramDiskVirtualDiskPersisGUID = [int('0x5cea02c9', 16), int('0x4d07', 16), int('0x69d3', 16), int('0x26', 16), int('0x9f', 16), int('0x44', 16), int('0x96', 16), int('0xfb', 16), int('0xe0', 16), int('0x96', 16), int('0xf9', 16)]
-        ramDiskVirtualCDPersisGUID = [int('0x08018188', 16), int('0x42cd', 16), int('0xbb48', 16), int('0x10', 16), int('0x0f', 16), int('0x53', 16), int('0x87', 16), int('0xd5', 16), int('0x3d', 16), int('0xed', 16), int('0x3d', 16)]
+        volitileMemGUID = [int('0x7305944f', 16), int('0xfdda', 16), int('0x44e3', 16), int('0xb1', 16), int('0x6c', 16),
```

Comment:
Why are we parsing ints here? Can't we just change `int('0x7305944f', 16)` to `0x7305944f`?

Reply:
You were the last person to touch this code section. :p  It could.

Reply:
In my defense, I didn't look at the file, just formatted it. 😇